### ISSUE TITLE
Workbench doesn't respect excluded_paths from .shiftrc

### ISSUE THREAD

**Issue by mortenscheel** [laravel-shift/workbench-issues#29]
**Desktop:**
 - OS: Ubuntu
 - OS Version: 20-04
 - App Version: 0.4.6.0

**Describe the bug**
Apparently Workbench doesn't read `excluded_paths` from `.shiftrc` when running tasks.

I've added `excluded_paths = "app/Subfolder"` to `.shiftrc` because the folder contains some problematic classes that trigger an infinite loop/recursion when PhpStan tries to analyze them. I can process the project with Rector, as long as this folder is ignored.

Is there some other way to get Workbench to ignore certain files or folders?

**To Reproduce**
1. Add a folder path (containing files that would be modified by the task) to `excluded_paths` in `.shiftrc`
2. Run the task
3. Check if any files in the excluded folder are modified

**Comment by jasonmccreary** [comment]
@mortenscheel, does the path you are specifying contain the `.shiftrc` file? Which task are you running? If it is a PHP task, what is output for the tasks for "scanning folder"?

**Comment by mortenscheel** [comment]
Hi @jasonmccreary 

No, the `.shiftrc` file is in the project root, and I'm trying to exclude `app/Subfolder`.
I'm running `php/switch-to-match`
The task log shows `>>> Event: Processing paths: database, app, config, routes, tests`.

By the way I finally figured out why PHPStan went into infinite recursion when trying to analyse those classes: phpstan/phpstan#5390

**Comment by jasonmccreary** [comment]
@mortenscheel, I assume you specify the path to the project root in the Workbench?

**Comment by mortenscheel** [comment]
Yes. I select the project folder using the Browse button, and it looks correct in the editor:
```yaml
path: /home/morten/Code/lv-tmp
tasks:
- name: 'switch to match'
  uses: php/switch-to-match
```

Besides, it does manage to modify some of the files, until it reaches the folder that causes infinite recursion (which should have been excluded).

I've tried the same recipe in Linux, Windows and online, and all of them fail. Presumably because they run out of memory trying to analyse the problematic folder.

**Comment by jasonmccreary** [comment]
Sounds good. We'll get this patched up in the next few days.

**Comment by jasonmccreary** [comment]
This should be patched in the latest release (0.4.7). Let us know if you have any issues.
